### PR TITLE
SONARJS-529 Add SQALE to S1451 and S1067 rules

### PR DESCRIPTION
**Pull request by vilchik-elena**

### CODE REVIEW

File: javascript-checks/src/main/java/org/sonar/javascript/checks/ExpressionComplexityCheck.java [hunk 1]
```diff
@@ -120,10 +126,10 @@ public void leaveNode(Tree tree) {
...
 
       if (currentExpression.isOnFirstExprLevel()) {
-        if (currentExpression.getExprNumberOfOperator() > max) {
-          addIssue(
-            tree,
-            "Reduce the number of conditional operators (" + currentExpression.getExprNumberOfOperator() + ") used in the expression (maximum allowed " + max + ").");
+        int complexity = currentExpression.getExprNumberOfOperator();
+        if (complexity > max) {
+          String message = String.format(MESSAGE, complexity, max);
+          getContext().addIssue(this, tree, message, complexity - max);
```
Comment (SonarTech) [inline javascript-checks/src/main/java/org/sonar/javascript/checks/ExpressionComplexityCheck.java]: MAJOR Cast one of the operands of this substraction operation to a "double". [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2184)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: javascript-checks/src/main/java/org/sonar/javascript/checks/FileHeaderCheck.java [hunk 2]
```diff
@@ -40,7 +43,8 @@
   key = "S1451",
   name = "Copyright and license headers should be defined",
   priority = Priority.BLOCKER)
-//@ActivatedByDefault
+@SqaleSubCharacteristic(RulesDefinition.SubCharacteristics.SECURITY_FEATURES)
+@SqaleConstantRemediation("5min")
 @NoSqale
```
Comment (pynicolas) [inline javascript-checks/src/main/java/org/sonar/javascript/checks/FileHeaderCheck.java]: @vilchik-elena Shouldn't we remove this line?

Comment (vilchik-elena) [inline javascript-checks/src/main/java/org/sonar/javascript/checks/FileHeaderCheck.java]: @pynicolas indeed!

**Pull request merged**